src/uu/numfmt/src/numfmt.rs

```diff
@@ -281,7 +281,22 @@ mod tests {
         }
     }
 
-    fn get_options() -> NumfmtOptions {
+    fn get_valid_options() -> NumfmtOptions {
+        NumfmtOptions {
+            transform: TransformOptions {
+                from: Unit::None,
+                to: Unit::None,
+            },
+            padding: 10,
+            header: 0,
+            fields: vec![Range { low: 0, high: 1 }],
+            delimiter: None,
+            round: RoundMethod::Nearest,
+            suffix: None,
+        }
+    }
+
+    fn get_invalid_options() -> NumfmtOptions {
```

Comment:
a warning here:

function is never used: `get_invalid_options`

Reply:
removed now :)